src/tests/functional/inference_engine/ir_serialization/custom_ops.cpp

```diff
@@ -13,7 +13,7 @@
 #include "ie_core.hpp"
 #include "ngraph/ngraph.hpp"
 #include "openvino/runtime/core.hpp"
-#include "transformations/serialize.hpp"
+#include "ngraph/pass/serialize.hpp"
```

Comment:
Why in some files included as `openvino` and as `ngraph` in others? 
Will be unified in phase 4?

Reply:
The task applies to transformation component only. Only necessary changes to other parts are made. `ov` namespace has precedence over the `ngraph` - that's my approach, but in this particular case it wouldn't compile. Note that `transformations/serialize.hpp` included `ngraph/pass/serialize.hpp`, that was its only purpose - being proxy, and was removed.